examples/v2/chat2.nim

```diff
@@ -66,6 +66,11 @@ proc dialPeer(c: Chat, address: string) {.async.} =
   discard await c.node.switch.dial(remotePeer, WakuRelayCodec)
   c.connected = true
 
+proc connectToNodes(c: Chat, nodes: openArray[string]) =
```

Comment:
@oskarth can we somehow make the one in node accessible. So we have some code reuse. One of the issues I had with debugging was that half the functions we had there were copied into here.

Reply:
Not in this PR.